entries/jQuery.widget.xml

```diff
@@ -200,6 +215,43 @@
 			<xi:include href="../includes/widget-method-instance.xml" xmlns:xi="http://www.w3.org/2003/XInclude"/>
 			<xi:include href="../includes/widget-method-option.xml" xmlns:xi="http://www.w3.org/2003/XInclude"/>
 			<xi:include href="../includes/widget-method-widget.xml" xmlns:xi="http://www.w3.org/2003/XInclude"/>
+			<method name="_addClass">
+				<desc>
+					Add classes to an element of the widget.
+					<p>This provides a hook for the user to add additional classes or replace default styling classes, through the <a href="#option-classes"><code>classes</code> option</a>.</p>
+					<p>It also provides automatic removal of these classes when a widget is destroyed.</p>
+				</desc>
+				<argument name="element" type="jQuery" optional="true">
+					<desc>The element to add the classes to. Defaults to <code>this.element</code>.</desc>
+				</argument>
+				<argument name="keys" type="String">
+					<desc>
+						The classes to add. If a property of the <code>classes</code> option matches a key, the value will be added as well.
+						<p>Like the <code>element</code> argument, this argument is also optional. To skip it, specify <code>null</code>.</p>
```

Comment:
Perhaps the note isn't necessary since we explicitly mention this in the description for `extra`.